packages/projects/src/views/Browse.jsx

```diff
@@ -71,9 +85,24 @@ const Browse = () => {
       const projects = await entityFactory({ resource })
       const items = await projects.getAll()
       dispatch({ type: 'items', payload: items })
+
+      const groupsApi = await entityFactory({ resource: 'group' })
+      const groups = await groupsApi.getAll()
+      dispatch({ type: 'groups', payload: groups })
+
+      const { queryService } = await services()
+      try {
+        const { profiles } = await queryService.recommend()
+        dispatch({ type: 'profiles', payload: profiles })
+        console.log(profiles)
+      } catch (error) {
+        console.log(error)
+      }
     })()
   }, [services])
 
+  const profiles = profilesByOwner(state.profiles)
```

Comment:
So state is a variable that can change and React keeps track of these state changes to know what to re-render. By not depending on state but extracting a value from it (profiles), it may not re-render properly when passed into the component. 

This is different from using the state hook and can be a bit non-intuitive at first. As a general rule, just pass the state to the component and extract the value in the component or depend directly on state in the main component.